src/util/time.cpp

```diff
@@ -57,11 +57,6 @@ std::chrono::seconds GetMockTime()
     return g_mock_time.load(std::memory_order_relaxed);
 }
 
-int64_t GetTimeMillis()
```

Comment:
27594: original PR also dropped
```
template <typename T>
static T GetSystemTime()
...
```
should be `partial`

Reply:
marked as partial